contracts/strategies/pangolin/strategy-png-avax-lost.sol

```diff
@@ -0,0 +1,124 @@
+pragma solidity ^0.6.7;
+
+import "../strategy-png-minichef-farm-base.sol";
+
+/// @notice The strategy contract for Pangolin's AVAX/LOST Liquidity Pool with PNG and LOST rewards
+contract StrategyPngAvaxLost is StrategyPngMiniChefFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public _poolId = 105;
+    address public png_avax_lost_lp = 0x8461681211B49c15e20B3Cfd4c63BE258878B7D9;
+    
+    address public lost = 0x449674B82F05d498E126Dd6615a1057A9c088f2C;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyPngMiniChefFarmBase(
+            _poolId,
+            png_avax_lost_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    /// @notice **** State Mutations ****
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        IMiniChef(miniChef).harvest(poolId, address(this));
+
+        /// @dev Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)  
```

Comment:
I don't know if NatSpec tags are supposed to be used as regular comments. I think they're meant to be headers for things like functions and interfaces. I would ask @Auroter for guidance on this

Reply:
I didn't add that comment, but I could remove it

Reply:
I think if you change them to regular comments it will be fine